### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by robxnsingh**
Added few web tools and chrome extension.

P.S- I actually added 4 of each tools and extensions, but by mistake on main branch. So in the 2nd attempt I did it in another branch(by deleting one of 4 each tools and links).

Thank you for reading my PR. Grateful!

### CODE REVIEW

File: chrome-extensions.md [hunk 1]
```diff
@@ -92,4 +92,8 @@
...
+[Wappalyzer](https://chrome.google.com/webstore/detail/wappalyzer-technology-pro/gppongmhjkpfnbhagpmjfkannfbllamg) | Wappalyzer is an effective tool to help find out all the underlying technologies used to build any website.
+[Colorzilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp)| Colorzilla is an eyedropper tool with a color picker, color history, CSS gradient analyzer, and every other color-related tool you can think of. It is a highly useful Chrome Extension for Web Developers to directly pick up the color and use it in their design or work.
+[check-my-links](https://chrome.google.com/webstore/detail/check-my-links/ojkcdipcgfaekbeaelaapakgnjflfglf)| Just as the name suggests, Check My Link crawls through your web page to show if it has broken links and checks for valid and redirected links It can be easily used and can be very helpful for maintaining the SEO standards of your webpage.
```
Comment (lucasgdb) [inline chrome-extensions.md:99]: all these tools were already added. Could you remove these changes, please?

File: web-tools.md [hunk 2]
```diff
@@ -190,4 +190,9 @@
...
 [Confluence](https://www.atlassian.com/software/confluence) | Confluence serves in the form of the web-based corporate Wiki developed by the Australian software company Atlassian.
 [CSS box shadow generator](https://cssgenerator.org/box-shadow-css-generator.html) | A box-shadow CSS generator that helps you quickly generate box-shadow CSS declarations for your website.
-[QRCode Generator](https://qrcode-generator.lucasbittencourt.dev/) | A website to generate customized QRCodes. ✨ Open Source ✨
+[QRCode Generator](https://qrcode-generator.lucasbittencourt.dev/) | A website to generate customized QRCodes. 
+[aciTIME](https://www.actitime.com/timesheet-for-productivity) | An intelligent timesheet tool, actiTIME, can be of great help for developers and their managers in attaining scope management and provide thorough reports.
+[Total](https://total.wpexplorer.com/) | A WordPress theme that is complete, easy to configure and customize, lightning-fast, affordable , and which will make your website convert better.
+[Startup-stash](https://startupstash.com/) | Startup Stash is the world’s largest online directory of tools and resources for startups and entrepreneurs.
+
+✨ Open Source ✨
```
Comment (lucasgdb) [inline web-tools.md:198]: could you fix this? this "✨ Open Source ✨" is from QRCode Generator tool.

**Review by lucasgdb — CHANGES_REQUESTED** [review]

File: chrome-extensions.md (hunk 1 above)
Comment (robxnsingh) [reply chrome-extensions.md:99]: Okay, I'll remove them!

File: web-tools.md (hunk 2 above)
Comment (robxnsingh) [reply web-tools.md:198]: Sure, I'll do it.

File: chrome-extensions.md (hunk 1 above)
Comment (lucasgdb) [reply chrome-extensions.md:99]: taking a look on the list of tools (pressing ctrl + F and search for it)

**Pull request closed without merging**